### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by jonmallVu**
Updated documentation for 1.170.1 release. Addresses limits/items mismatch @SamMiners.

### CODE REVIEW

File: docs/source/DeveloperDocumentation/TaskEngineWorkflows.md [hunk 1]
```diff
@@ -124,12 +127,12 @@ This workflow allows you to create a frame accurate VOD clip by passing in a sta
...
-| clip: key_id      |No | Should the stream be DRM’d we would require the KeyID. ||
-| clip: content_key |No | Should the stream be DRM’d we would require the Content Key. ||
+| clips.source      |Yes| This would need to be either an HLS, MSS or Dash stream URL to the Live or Archive content. e.g. `http://mydomain.com/test.ism/.m3u8` , `http://mydomain.com/test.ism/manifest` , `http://mydomain.com/test.ism/.mpd` ||
+| clips.start       |No | UTC timestamp for the start timecode. e.g 2016-10-13T10:10:40.251Z or Offsets e.g. “hh:mm:ss”. ||
+| clips.end         |No | UTC timestamp for the end timecode e.g 2016-10-13T10:20:40.251Z or Offsets e.g. “hh:mm:ss”. ||
+| clips.filter      |No | This allows you to pass filter expressions to select certain video, audio tracks. e.g. to all video bitrates below 8Mbps and all audio bitrates at 64Kbps "type==\\"video\\"&&systemBitrate==800000\|\|type==\\"audio\\"&&systemBitrate==64000". ||
+| clips.key_id      |No | Should the stream be DRM’d we would require the KeyID. ||
```
Comment (samuelbewick) [inline docs/source/DeveloperDocumentation/TaskEngineWorkflows.md:134]: > be

File: docs/source/ReleaseNotes.md [hunk 2]
```diff
@@ -1,72 +1,104 @@
...
+
+###### VOD Remix
+
+- Added support for creating AVOD playlists with SCTE35 ad insertion and replacement markers.
+- Added support for creating live streams from VOD content with SCTE35 ad replacement markers.
+
+###### Asset Delete
+
+- Bugfix: Added checks for empty arrays before deleting attempting to delete files.
+-
```
Comment (samuelbewick) [inline docs/source/ReleaseNotes.md:41]: Empty bullet point.

**Review by samuelbewick — APPROVED** [review]
I notice use of "event" in the terminology a lot, which arguably isn't a TaskEngine concept specifically so this documentation doesn't do quite as well by itself as it would otherwise. Perhaps that's the intention, especially as it's getting less and less likely that a client only has the TE?

File: docs/source/ReleaseNotes.md (hunk 2 above)
Comment (jonmallVu) [reply docs/source/ReleaseNotes.md]: Good spot! Thanks.

**Comment by jonmallVu** [comment]
> I notice use of "event" in the terminology a lot, which arguably isn't a TaskEngine concept specifically so this documentation doesn't do quite as well by itself as it would otherwise. Perhaps that's the intention, especially as it's getting less and less likely that a client only has the TE?

I understand your point. Event in the TE context is used to encompass any form of VOD (captured, ingested, remix, simulated live) etc. There shouldn't be reference to an event within the VCH context in the doc , it's more the use of the generic noun event.

File: docs/source/ReleaseNotes.md [hunk 3]
```diff
@@ -1,72 +1,104 @@
...
-* [v1.167](#v1-167)
-* [v1.166](#v1-166)
-* [v1.165](#v1-165)
+- [v1.171.x](#v1171x)
+- [v1.170.x](#v1170x)
+- [v1.169.x](#v1169x)
+- [v1.168.x](#v1168x)
+- [v1.167.x](#v1167x)
+- [v1.166.x](#v1166x)
+- [v1.165.x](#v1165x)
```
Comment (RoozyRoo) [inline docs/source/ReleaseNotes.md:9]: Should there be a Hyphen where the dots are for the bracketed versions? E.G (#v1-169-x) Tested the Docs https://docs.vualto.com/projects/task-engine/en/dev-branch/ReleaseNotes.html and clicking them doesn't interact in any way currently.

**Review by RoozyRoo — CHANGES_REQUESTED** [review]

File: docs/source/ReleaseNotes.md (hunk 3 above)
Comment (jonmallVu) [reply docs/source/ReleaseNotes.md:9]: Ehh thanks Pete... the MD extension I'm using keeps removing them 🙄 .

**Review by RoozyRoo — APPROVED** [review]

**Pull request merged**